.github/workflows/prettier.yml

```diff
@@ -0,0 +1,23 @@
+name: DCR prettier 💅
+on:
+    push:
+        paths-ignore:
+            - "apps-rendering/**"
+
+jobs:
+    lint:
+        name: DCR Prettier
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: guardian/actions-setup-node@main
+
+            # Cache npm dependencies using https://github.com/bahmutov/npm-install
+            - uses: bahmutov/npm-install@v1
```

Comment:
FYI, I think `guardian/actions-setup-node` is able to handle this directly now? @sndrs pulled in the upstream changes that add support for caching.